source/_integrations/tplink_eap.markdown

```diff
@@ -0,0 +1,38 @@
+---
+title: TP-Link EAP
+description: Instructions on how to use a TP-Link EAP access point as a device tracker.
+logo: tp-link.jpg
+ha_category:
+  - Presence Detection
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_release: 0.117
+ha_domain: tplink_eap
+---
+
+This platform allows you to detect presence by looking at devices connected to a [TP-Link EAP](https://www.tp-link.com/us/business-networking/ceiling-mount-access-point/).
+
+<div class='note warning'>
+This integration uses the web interface to retrieve data from the device. The device webserver does not support concurrent connection.
+If you try to login to the interface, your session will be terminated when the integration will poll the device status.
+<br>
```

Comment:
Please use markdown as much as possible.

Reply:
Could I have some guidance for that piece?
I was thinking this fell under the valid use of HTML according to https://developers.home-assistant.io/docs/documenting/create-page#html

My rationale for using the HTML: I feel that this is somewhat of importance to users of that integration, so I would be a bit reluctant to not have that piece of information highlighted in some way.